pkg/nodejs/npm/parse.go

```diff
@@ -167,6 +205,26 @@ func findDependsOn(pkgPath, depName string, packages map[string]Package) (string
 			return utils.PackageID(depName, dep.Version), nil
 		}
 	}
+
+	// There are cases when pkgPath doesn't have `node_modules` suffix (e.g. when it is local package)
+	// add `node_modules` and try to find dep
+	// e.g. for pkgPath == `function/func1`, depName == "debug`:
+	// `node_modules/debug`
+	// case when `function/func1/node_modules_debug` exists resolved in loop
```

Comment:
should there be a `function/func1/node_modules/debug` here?

Reply:
Right! It is typo.
I fixed this in https://github.com/aquasecurity/go-dep-parser/pull/209/commits/57989a6aa3f30ee56418ddffec957f42c438b68c